docs/src/guide/index.md

```diff
@@ -367,6 +384,43 @@ const knex = require('knex')({
 });
 ```
 
+#### Custom validation (`pool.validate`)
+
+You can provide a `pool.validate` function to decide if a pooled connection should be reused. It can be sync or async; return/resolve `false` (or throw) to veto reuse and force the pool to create a fresh connection.
```

Comment:
```suggestion
You can provide a `pool.validate` function to decide if a pooled connection should be reused. It can be sync or async; return/resolve `false` (or throw) to discard the connection and force the pool to replace it.
```

This is super minor, but the phrase "... to veto reuse and ..." could be misunderstood; the noun and verb forms have slightly differing connotations (the noun, as in "to veto the reuse of", is the intended meaning here, but the verb, as in "to reuse", means the opposite)